src/ui_utils.h

```diff
@@ -51,6 +51,8 @@ typedef struct GeanyInterfacePrefs
 	gchar			*tagbar_font;				/**< symbol sidebar font */
 	gchar			*msgwin_font;				/**< message window font */
 	gboolean		show_notebook_tabs;			/**< whether editor tabs are visible */
+	gboolean		auto_sort_tabs_filename;	/**< whether to automatically sort tabs based on filename */
+	gboolean		auto_sort_tabs_pathname;	/**< whether to automatically sort tabs based on pathname */
```

Comment:
need to be added to the end of the structure not to break ABI, as the other fields are part of the API.
Also, those new fields should probably not be added to the API (`/**` comment), so make the comment use `/*` only.